calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/views/initialization_view.py

```diff
@@ -75,6 +85,49 @@ def __init__(self, calibrator: "CameraIntrinsicsCalibratorUI", cfg):  # noqa F82
         source_layout.addWidget(self.data_source_combobox)
         self.source_group.setLayout(source_layout)
 
+        self.params_combobox = QComboBox()
+        self.params_combobox.addItem("General", 0)
+        self.params_combobox.addItem("C1", 1)
+        self.params_combobox.addItem("C2", 2)
+        self.params_combobox.addItem("Ceres Calib", 3)
+        self.params_combobox.addItem("Load File", 4)
+
+        def on_params_combo_box_changed(index):
+            if self.params_combobox.currentText() == "Load File":
+                file_name, _ = QFileDialog.getOpenFileName(
+                    self, "Open File", "", "All Files (*.*);;Text Files (*.yaml)"
+                )
+                if file_name:
+                    print(f"Selected file: {file_name}")
+                    config_file_path = file_name
+            elif self.params_combobox.currentText() == "C1":
+                config_file_path = os.path.join(config_dir, "c1_intrinsics_calibrator.yaml")
+            elif self.params_combobox.currentText() == "C2":
+                config_file_path = os.path.join(config_dir, "c2_intrinsics_calibrator.yaml")
+            elif self.params_combobox.currentText() == "Ceres Calib":
+                config_file_path = os.path.join(config_dir, "intrinsics_calibrator_ceres.yaml")
+            elif self.params_combobox.currentText() == "General":
+                config_file_path = os.path.join(config_dir, "intrinsics_calibrator.yaml")
+
+            if config_file_path:
+                cfg = {}
+                try:
+                    with open(config_file_path, "r") as stream:
+                        cfg = yaml.safe_load(stream)
+                        self.cfg = defaultdict(dict, cfg)
+                        self.update_board_type()
+                        print("successfully loaded")
+                except Exception as e:
+                    logging.error(f"Could not load the parameters from the YAML file ({e})")
+
+        self.params_combobox.currentIndexChanged.connect(on_params_combo_box_changed)
+
+        self.params_group = QGroupBox("Parameters Profile")
```

Comment:
let's not mix cases

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e

Reply:
This was not addressed

Reply:
https://github.com/tier4/CalibrationTools/commit/f89b3ff4cb6f41ae9f344962700594a3b0d2fee4